sdk/debugger_interface/linux/gdb-sgx-plugin/gdb_sgx_plugin.py

```diff
@@ -708,6 +708,37 @@ def stop(self):
         GetMuslLoadLibraryReturnBreakpoint()
         return False
 
+class GetGlibcMapObjectReturnBreakpoint(gdb.FinishBreakpoint):
```

Comment:
Can we merge the GetMuslLoadLibraryReturnBreakpoint and GetGlibcMapObjectReturnBreakpoint?

Reply:
After a review of GetMuslLoadLibraryReturnBreakpoint, we can merge them